client/src/shared/ui/overlay/OverlayDropdown.js

```diff
@@ -118,6 +120,7 @@ export function OverlayDropdown(props) {
 function OptionGroup(props) {
   const {
     items,
+    insert,
```

Comment:
One thing I forgot to mention in our sync: maybe `labelSuffix` is a better term here?  To name it as a generic prop that always renders _after_ the label / title.